doc/doxygen/Doxyfile.in

```diff
@@ -435,13 +435,13 @@ LOOKUP_CACHE_SIZE      = 0
 # normally produced when WARNINGS is set to YES.
 # The default value is: NO.
 
-EXTRACT_ALL            = NO
+EXTRACT_ALL            = YES
 
 # If the EXTRACT_PRIVATE tag is set to YES, all private members of a class will
 # be included in the documentation.
 # The default value is: NO.
 
-EXTRACT_PRIVATE        = NO
+EXTRACT_PRIVATE        = YES
```

Comment:
I currently use the doxygen documentation also as a developer to help me recall details about my own implementation. Then it is useful to see private members as well.

Reply:
Hmm. The thing is that if we check this change in, then it will also appear in the ["online doxygen documentation"](https://craffael.github.io/lehrfempp/) that is automatically generated. How about you keep this change on your local machine but don't check it into git?

Reply:
I would not do that because then you will not be able to make further changes to this file and commit them. Instead I would just omit Doxyfile.in manually from any commit...

Reply:
The problem is, that if you check this in, then it is rather hard to commit further changes for `Doxyfile.in`. Therefore I suggest that you just omit Doxyfile.in from further commits manually.